packages/treasury/bundles/install-on-chain.js

```diff
@@ -74,42 +88,65 @@ export async function installOnChain({
     }),
   );
 
+  const electorateTerms = { committeeName: 'TreasuryBoard', committeeSize: 5 };
```

Comment:
Where did we get 5 for the size of the treasury board? Was that specified somewhere? Or is it arbitrary?

Reply:
It was arbitrary.  Can you think of a good place to put a parameter/manifest constant?

Reply:
I'm not sure a manifest constant would help me; then I'd just wonder why _that_ is 5. I'm trying to learn the story behind these numbers and structures. Which ones stay fixed and which ones change and how and why? I'm struggling to pick it up from the code alone.

Reply:
a do-nothing electorate would make that constant unnecessary.